formidable/forms/field_builder.py

```diff
@@ -89,11 +89,9 @@ def get_help_text(self):
         return self.field.help_text
 
     def get_validators(self):
```

Comment:
I agree on build_rules too.
If I want all the validators as the name ``get_validators`` suggests, I need to cast it to list() for instance in order to have them all in python2. I would change it for gen_validators and gen_rules (for generator)